pytorch_lightning/trainer/trainer.py

```diff
@@ -954,6 +954,9 @@ def run_evaluation(self, on_epoch: bool = False) -> _EVALUATE_OUTPUT:
                 if batch_idx >= dl_max_batches:
                     break
 
+                # TODO: where is the profile step for to_device?
+                batch = self.accelerator.to_device(batch)
+
```

Comment:
does the same apply for the predict loop?

Reply:
Yes, will continue the PR shortly. Not sure if this will be correct, havent checked in detail.

Reply:
done